sdk/go/common/workspace/project.json

```diff
@@ -296,21 +299,6 @@
             "title":"ConfigTypeDeclaration",
             "type":"object",
             "additionalProperties":false,
-            "required":[
-                "type"
-            ],
-            "if":{
```

Comment:
Are we treating `type: array` without items as array<any> then?

Reply:
I've removed because it trips up the validation when only defining `default` property. It says "`if-then` failed because there is no `items` property" but I didn't even specify what the `type` is. 

I've added validation when loading the project to achieve the same result.